Commit and close my branch

Zielous opened the pull request:
@githubteacher closes #6093

brianamarie commented:
Hi @Zielous! So that the tests pass, make sure that:
- The `---` is on the last line in addition to the first line
- There is a space after each `:` on lines 2-4

If these changes haven't been made by May 12, the GitHub training team may make the changes for you so we can get your branch merged in. Let me know if you need or want any help!

Zielous commented:
@brianamarie Thanks for that! When the first check failed for my first commit, it said "warning  too many spaces before colon  (colons)" so I removed the spaces after the colons, but this error persisted for the second check after I committed, so I was a bit confused.

I didn't get an error in either commits about adding the "---" to the end of the file (I added it at the start only), so couldn't figure this part out.

Anyway, thanks for the help! :D

brianamarie commented:
@Zielous Awesome! Yeah, you're definitely on the right track. Let me know if there's anything more I can help with.
